onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -808,5 +808,22 @@ def testRunModelWithCudaCopyStream(self):
             for iteration in range(100000):
                 result = session.run(output_names=['output'], input_feed={'shape': shape})
 
+    def testSharedAllocatorUsingCreateAndRegisterAllocator(self):
+        # Create and register an arena based allocator
+        ort_arena_cfg = onnxrt.OrtArenaCfg(0, -1, -1, -1)
+        ort_memory_info = onnxrt.OrtMemoryInfo("Cpu", onnxrt.OrtAllocatorType.ORT_ARENA_ALLOCATOR, 0, onnxrt.OrtMemType.DEFAULT)
+        onnxrt.create_and_register_allocator(ort_memory_info, ort_arena_cfg)
```

Comment:
nit: can the user choose to not pass OrtArenaCfg at all when calling create_and_register_allocator? Can choose to address in a separate PR.

Reply:
This is a good point - this is because the signature of the global method takes in a reference and not a pointer. Hmm - I wonder if pybind will complain if I make it a pointer. I would think not. Let me try that.

Reply:
Fixed